src/amo/components/RatingsByStar/index.js

```diff
@@ -86,85 +90,104 @@ export class RatingsByStarBase extends React.Component<InternalProps> {
     );
   }
 
+  handleRouting(star: string) {
+    const { addon, location, history, clientApp, siteLang } = this.props;
+    history.push({
+      pathname: `/${siteLang}/${clientApp}/addon/${addon.slug}/reviews/`,
+      search: `?score=${star}&src=${location.query.src}`,
```

Comment:
If `src` does not exist on the URL, then this yields a link with `src=undefined` in it, which I don't think we want.

Reply:
Yes, we should add a conditional check for `src`. I will update that.